test/parallel/test-icu-data-dir.js

```diff
@@ -8,7 +8,7 @@ const { spawnSync } = require('child_process');
 
 const expected =
     'could not initialize ICU (check NODE_ICU_DATA or ' +
-    '--icu-data-dir parameters)' + (common.isWindows ? '\r\n' : '\n');
+    `--icu-data-dir parameters)${common.isWindows ? '\r\n' : '\n'}`;
```

Comment:
@refack Sorry, I can't get your meanings. Do you mean the code after`const expected ...` to be one line?